org.sdrangel.SDRangel.json

```
@@ -0,0 +1,525 @@
+          "commit": "afefa23"
+        }
+      ]
+    },
+    {
+      "name": "xtrx",
+      "buildsystem": "cmake-ninja",
+      "subdir": "sources",
+      "config-opts": [
+        "-Wno-dev",
+        "-DENABLE_SOAPY=NO"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/xtrx-sdr/images.git",
+          "commit": "9586a6e"
+        }
+      ]
+    },
+    {
+      "name": "uhd",
+      "buildsystem": "cmake-ninja",
+      "subdir": "host",
+      "config-opts": [
+        "-Wno-dev",
+        "-DENABLE_PYTHON_API=OFF",
+        "-DINSTALL_UDEV_RULES=OFF"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://github.com/EttusResearch/uhd.git",
+          "commit": "e0e61a5"
+        },
+        {
+          "type": "patch",
+          "path": "uhd-disable-ascii-art-dft.patch"
+        },
+        {
+          "type": "patch",
+          "path": "uhd-disable-latency-utils.patch"
+        }
+      ]
+    },
+    {
+      "name": "libmirisdr",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://github.com/f4exb/libmirisdr-4.git"
+        }
+      ]
+    },
+    {
+      "name": "soapy",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://github.com/pothosware/SoapySDR.git",
+          "commit": "5838bc9"
+        }
+      ]
+    },
+    {
+      "name": "soapy_remote",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://github.com/pothosware/SoapyRemote.git",
+          "commit": "4f5d717"
+        }
+      ]
+    },
+    {
+      "name": "soapy_limesdr",
+      "// TODO": "Basis is contained in build modules limesdr and soapy_remote",
+      "// buildsystem": "cmake-ninja",
+      "// config-opts": [
+        "-Wno-dev"
+      ]
+    },
+    {
+      "name": "soapy_uhd",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/pothosware/SoapyUHD.git",
+          "commit": "2900fff"
+        }
+      ]
+    },
+    {
+      "name": "soapy_redpitaya",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/pothosware/SoapyRedPitaya.git",
+          "commit": "3d576f83b3bde52104b2a88150516ca8c9a78c7a"
+        }
+      ]
+    },
+    {
+      "name": "sdrangel",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Wno-dev",
+        "-DDEBUG_OUTPUT=ON",
+        "-DCMAKE_BUILD_TYPE=RELEASE",
```

```suggestion
        "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
```

this allow having debuginfo (split out in the .Debug) for debugging.